scripts/build-n8n.mjs

```diff
@@ -233,17 +270,23 @@ echo(chalk.green.bold('================ BUILD SUMMARY ================'));
 echo(chalk.green(`✅ n8n built successfully!`));
 echo('');
 echo(chalk.blue('📦 Build Output:'));
+echo(chalk.green('   n8n:'));
 echo(`   Directory:      ${path.resolve(config.compiledAppDir)}`);
 echo(`   Size:           ${compiledAppOutputSize}`);
 echo('');
+echo(chalk.green('   task-runner-javascript:'));
```

Comment:
I hope we don't have any automation that cares about the exact format of this output 😅